From f4fd89d643a07b92db57591c02ff72c46664b607 Mon Sep 17 00:00:00 2001
From: Simeon Marijon <simeon.marijon@foss.st.com>
Date: Tue, 20 Jun 2023 14:37:00 +0200
Subject: [PATCH] arm64: defconfig: add stm32 backup registers reboot mode
 support in addons fragment

Enable REBOOT MODE from nvmmem cells with
fragment-02-defconfig-addons.config (STM32MP25 Socs)

Change-Id: I30f0d332e4f73f1c4a567e6b6719b279d503be9f
Signed-off-by: Simeon Marijon <simeon.marijon@foss.st.com>
(cherry picked from commit b4ebd41f4fe70429c8ea8a729c0e0c64b4977275)
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index bc1e5fd2502b..0fb8fd3b0a3e 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -14,6 +14,7 @@ CONFIG_MTD_NAND_STM32_FMC2=y
 CONFIG_MTD_HYPERBUS=y
 CONFIG_MTD_SPI_NAND=y
 CONFIG_MTD_UBI=y
+CONFIG_NVMEM_REBOOT_MODE=y
 CONFIG_NVMEM_STM32_ROMEM=y
 CONFIG_NVMEM_STM32_TAMP=y
 CONFIG_OF_OVERLAY=y
-- 
2.39.5

